packages/serverless/src/awslambda.ts

```diff
@@ -46,6 +47,7 @@ export interface WrapperOptions {
   callbackWaitsForEmptyEventLoop: boolean;
   captureTimeoutWarning: boolean;
   timeoutWarningLimit: number;
+  captureAllSettledReasons: boolean;
```

Comment:
Would be great if you could leave a comment about what `captureAllSettledReasons` does, and it's default value.

Reply:
Added in ff5d46e12493b7655c8d6fbda66d46fad0f6fd01

Reply:
@AbhiPrasad : I create a PR for `sentry-doc` there https://github.com/getsentry/sentry-docs/pull/4374 ;)